rs/rosetta-api/icp/BUILD.bazel

```diff
@@ -74,7 +74,6 @@ DEV_DEPENDENCIES = [
     "//rs/ledger_suite/icrc1",
     "//rs/ledger_suite/icrc1/test_utils",
     "//rs/ledger_suite/icrc1/tokens_u256",
-    "//rs/nns/governance",
```

Comment:
This is the ONLY non-governance code that depends on //rs/nns/governance. To prevent anyone else from depending on //rs/nns/governance, I have reduced its visibility. See rs/nns/governance/BUILD.bazel.